src/Caesar.java

```diff
@@ -0,0 +1,28 @@
+import java.util.Scanner;
+
+public class Caesar {
+
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+        System.out.print("Text eingeben: ");
+        String text = sc.nextLine();
+        int translation = (int) (Math.random() * 26) + 1;
+        System.out.print("Verschiebung um " + translation + " Stellen");
+        String ausgabe = "";
+
+        for (int i = 0; i < text.length(); i++) {
+
+            char letter = text.charAt(i);
```

Comment:
Ansich ist das schon recht korrekt allerdings bewegen wir uns hiermit auch in den Bereich von unlesbaren Zeichen (z.b. wenn du ein kleines 'z' um drei stellen verschiebst kommt kein 'c' raus sondern ein komisches Symbol. 

D.h. wir sollten hier nicht sicher stellen, dass wir uns immer im Buchstaben bereich bewegen - 'a' bis 'z' oder 'A' bis 'Z', alle Zeichen außerhalb wollen wir bei der Caesar Verschlüsselung eigentlich nicht verschlüsseln, also ein leerzeichen sollte ein leerzeichen bleiben, ein punkt sollte ein punkt bleiben.